### PR TITLE
Add Missing ServiceMonitors "additionalLabels" in Values File

### PR DESCRIPTION
**Pull request by Xtigyro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 * [ENHANCEMENT] Expose `client_max_body_size` config for nginx max request body size #137
 * [ENHANCEMENT] Adding option to add custom headers (ex. X-Scope-OrgID) to NGINX from values.yaml (key `nginx.config.setHeaders`). #127
 
+## 0.4.2 / 2021-05-03
+
+* [BUGFIX] Added missing ServiceMonitors `additionalLabels` placeholders in `values.yaml`.
```
Comment (ShuzZzle) [inline CHANGELOG.md:10]: Please add your bugfix under the master / unreleased section. Not here!

File: Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 0.4.1
+version: 0.4.2
```
Comment (ShuzZzle) [inline Chart.yaml:2]: I appreciate you helping us out, but the maintainers/owner will decide when to cut releases. So please remove that change

**Review by ShuzZzle — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Xtigyro) [reply CHANGELOG.md]: @ShuzZzle Done.

File: Chart.yaml (hunk 2 above)
Comment (Xtigyro) [reply Chart.yaml]: @ShuzZzle Done.

Comment (ShuzZzle) [reply Chart.yaml]: Dope I saw in the diff that one 0.3.0 enhancement had no PR number so I quickly added that :)

**Review by ShuzZzle — APPROVED** [review]
LGTM

**Pull request merged**